sample/src/main/java/com/airbnb/android/showkasesample/Rows.kt

```diff
@@ -186,6 +196,7 @@ fun ProductCardRow(imageUrl: String, title: String, subtitle: String) {
             )
 
         }
+        BackHandler(onBack = {  })
```

Comment:
maybe leave a comment so future readers know why this seemingly non functional code was added, and so it doesn't get deleted.

imo probably better to put it in an explicitly named sample